Dockerfile

```
@@ -1,5 +1,7 @@
 FROM gcr.io/distroless/cc
 
+ENV ITCHYSATS_ENVIRONMENT=docker
```

this would be slightly more conventional:

```suggestion
ENV ITCHYSATS_ENV=docker
```